### DISCUSSION TITLE
某个关于绳匠叠楼的问题

### DISCUSSION THREAD

**Discussion by bin-lin2** [share121/inter-knot#1537] [其他]
如图
![微信图片_20240811230255](https://github.com/user-attachments/assets/351d51cd-5fac-41b9-b0f2-ef8eb00c0e25)
这样的回复是无法出现在绳网上的

只是简单提个醒，因为我也这样做过

**Comment by FurinaShark** [comment]
感谢提醒

**Comment by khiqwq** [comment]
以前能看见叠楼，现在不行力
